rust/kcl-lib/src/std/assert.rs

```diff
@@ -23,49 +23,66 @@ async fn _assert(value: bool, message: &str, args: &Args) -> Result<(), KclError
 
 /// Check that the provided value is true, or raise a [KclError]
 /// with the provided description.
-pub async fn assert(_exec_state: &mut ExecState, args: Args) -> Result<KclValue, KclError> {
-    let (data, description): (bool, String) = args.get_data()?;
-    inner_assert(data, &description, &args).await?;
+pub async fn assert(exec_state: &mut ExecState, args: Args) -> Result<KclValue, KclError> {
+    let condition = args.get_unlabeled_kw_arg_typed("condition", &RuntimeType::bool(), exec_state)?;
+    let error: String = args.get_kw_arg("error")?;
```

Comment:
Should be string type, right? I.e. using `get_kw_arg_typed()`.

Reply:
This is what currently happens if `error` is wrong type: 
<img width="429" alt="Screenshot 2025-04-21 at 11 37 22 AM" src="https://github.com/user-attachments/assets/fe1f5038-02b3-43e4-9ce7-a6c055eca445" />

Reply:
Using the `typed` flavor function would allow us to coerce non-string KCL values to string, if we ever wanted that. I guess we don't currently do that so it doesn't matter.